parser/fileparser.go

```diff
@@ -182,7 +182,7 @@ func mapBroHeaderToParserType(header *fpt.BroHeader, broDataFactory func() pt.Br
 	for index, name := range header.Names {
 		lu, ok := fieldTypes[name]
 		if !ok {
-			//NOTE: an unmatched field which exists in the log but not the struct
+			//An unmatched field which exists in the log but not the struct
```

Comment:
This is a well known behavior of RITA at this point. Perhaps it should be better documented elsewhere.